03-Node-npm-modules/movie.js

```diff
@@ -0,0 +1,27 @@
+var director = require('./director.js')
```

Comment:
He's using director local variable from the Class constructor not the Director class required from this line. Also if this is a class this should be Director instead of director